common/djangoapps/student/tasks.py

```diff
@@ -19,7 +19,7 @@
 log = logging.getLogger('edx.celery.task')
 
 
-@task(bind=True)
+@task(bind=True, name='student.send_activation_email')
```

Comment:
To avoid the rename, this should have been `student.tasks.send_activation_email`